scripts/generatePiImage.sh

```diff
@@ -3,7 +3,7 @@
 NEW_JAR=$(realpath $(find . -name photonvision\*-linuxarm32.jar))
 echo "Using jar: " $NEW_JAR
 sudo apt install xz-utils
-curl -sk https://api.github.com/repos/photonvision/photon-pi-gen/releases/tags/v2023.1.0-beta-1 | grep "browser_download_url.*xz" | cut -d : -f 2,3 | tr -d '"' | wget -qi -
+curl -sk https://api.github.com/repos/photonvision/photon-pi-gen/releases/tags/v2023.1.0-beta-2 | grep "browser_download_url.*xz" | cut -d : -f 2,3 | tr -d '"' | wget -qi -
```

Comment:
Actually probably needs us to cut a new release on the pi image gen repo from the aarch64 branch, as well as change the JAR name to linuxarm64